src/app/models/constraint-models/constraint.clone.spec.ts

```diff
@@ -1,30 +1,28 @@
-import {Constraint} from '../../models/constraint-models/constraint';
-import {ConstraintSerialiser} from './constraint-serialiser';
-import {ConstraintCopier} from './constraint-copier';
-import {TrueConstraint} from '../../models/constraint-models/true-constraint';
-import {ValueConstraint} from '../../models/constraint-models/value-constraint';
-import {TrialVisitConstraint} from '../../models/constraint-models/trial-visit-constraint';
-import {TimeConstraint} from '../../models/constraint-models/time-constraint';
-import {SubjectSetConstraint} from '../../models/constraint-models/subject-set-constraint';
-import {StudyConstraint} from '../../models/constraint-models/study-constraint';
-import {PedigreeConstraint} from '../../models/constraint-models/pedigree-constraint';
-import {NegationConstraint} from '../../models/constraint-models/negation-constraint';
-import {CombinationConstraint} from '../../models/constraint-models/combination-constraint';
-import {ConceptConstraint} from '../../models/constraint-models/concept-constraint';
-import {TrialVisit} from '../../models/constraint-models/trial-visit';
-import {Study} from '../../models/constraint-models/study';
-import {Concept} from '../../models/constraint-models/concept';
+import {Constraint} from './constraint';
+import {ConstraintSerialiser} from '../../utilities/constraint-utilities/constraint-serialiser';
+import {TrueConstraint} from './true-constraint';
+import {ValueConstraint} from './value-constraint';
+import {TrialVisitConstraint} from './trial-visit-constraint';
+import {TimeConstraint} from './time-constraint';
+import {SubjectSetConstraint} from './subject-set-constraint';
+import {StudyConstraint} from './study-constraint';
+import {PedigreeConstraint} from './pedigree-constraint';
+import {CombinationConstraint} from './combination-constraint';
+import {ConceptConstraint} from './concept-constraint';
+import {TrialVisit} from './trial-visit';
+import {Study} from './study';
+import {Concept} from './concept';
 
-describe('ConstraintCopier', () => {
+describe('Constraint.clone', () => {
```

Comment:
couldn't it be a part of constraint.spec class?

Reply:
Has been moved now.